tests/post/features/pods_alive.feature

```diff
@@ -7,3 +7,16 @@ Feature: Pods should be alive
     Scenario: Exec in Pods
         Given the Kubernetes API is available
         Then we can exec 'true' in the 'salt-master-bootstrap' pod in the 'kube-system' namespace
+
+    Scenario: Expected Pods
+        Given the Kubernetes API is available
+        Then we have 1 running pod labeled 'component=kube-controller-manager'
```

Comment:
Not very fond of hard-coding an expected 'number of nodes' (basically) in a test. This test won't be portable across single-node and multi-node test environments.

Reply:
Hmm what about creating a test case per host, I guess we can filter on the host name and determine how many pods there should be, or at least a control plane should have > 1 pod for each of the control plane components

Reply:
@NicolasT agree with you but for now, I don't know exactly what we should expect on multi-node environments and I can't find a more suitable test input. Unless you have suggestion, I'll let it as it was, and I'll update the test accordingly when we have a multi node environments

Reply:
For several of the Pods you look for, we want *at least one* instance. Don't really care about the actual count.

Reply:
Ok, thanks, I updated it to verify that we have at least one pods except for the dns 2